flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -298,7 +298,7 @@ class FlutterLocalNotificationsPlugin {
               id, title, body, scheduledDate, notificationDetails?.android,
               payload: payload,
               androidAllowWhileIdle: androidAllowWhileIdle,
-              matchDateComponents: matchDateTimeComponents);
+              matchDateTimeComponents: matchDateTimeComponents);
```

Comment:
Thanks for picking up on this. I'll need to bump the plugin more as this is actually a breaking change, which in itself is fine. I'll do this in another PR as there's another change I'd like to do before this would get released

Reply:
So i should include change? 

I also noticed i could rename the variable name `validateDateIsInTheFuture.matchDateTimeComponents`, should i do this?

Reply:
Which change are you referring to including? Normally, it's better to separate out these changes to another PR since one adds functionality whilst another introduces a breaking change to isolate the changes in case a rollback is needed but this seems straightforward enough that I'm ok with it.

Yep forgot to say `validateDateIsInTheFuture.matchDateTimeComponents` could be renamed as well but forgot this as I had left my original comments in the old PR that was closed

Reply:
Ok, i think i've fixed all the comments now.